playbooks/roles/prometheus/meta/main.yml

```diff
@@ -2,12 +2,23 @@
 dependencies:
   - name: ernestas-poskus.prometheus
     prometheus_node_exporter_install: false
+    prometheus_alert_manager_config_global:
+     smtp_smarthost: '{{ PROMETHEUS_ALERT_MANAGER_SMTP_SMARTHOST }}'
+     smtp_from: '{{ PROMETHEUS_ALERT_MANAGER_SMTP_FROM }}'
+     smtp_auth_username: '{{ PROMETHEUS_ALERT_MANAGER_SMTP_USERNAME }}'
+     smtp_auth_password: '{{ PROMETHEUS_ALERT_MANAGER_SMTP_PASSWORD }}'
   - name: nginx-proxy
     NGINX_PROXY_CONFIG_PATH: /etc/nginx/sites-available/prometheus
     NGINX_PROXY_PUBLIC_PORT: 443
     NGINX_PROXY_INTERNAL_PORT: 9090
     NGINX_PROXY_USERNAME: "{{ PROMETHEUS_BASIC_AUTH_USERNAME }}"
     NGINX_PROXY_PASSWORD: "{{ PROMETHEUS_BASIC_AUTH_PASSWORD }}"
+  - name: nginx-proxy
+    NGINX_PROXY_CONFIG_PATH: /etc/nginx/sites-available/alertmanager
+    NGINX_PROXY_PUBLIC_PORT: 9043
```

Comment:
So I can see it's not a practice shown here but we've been just putting a "1" in front of the internal port to get the public port. In the case of Grafana and Prometheus themselves I think it's just because we'll be accessing those pretty often. I don't see the same with the AlertManager that much, since alerts are usually turned off once the Prometheus alert rule expression is returning false again, so we may not be interacting with the GUI too much.

Reply:
Updated.

Reply:
Ops. I did not want to restart vagrant after updating it earlier so had changed it back for local testing and looks like it got caught.